package-lock.json

```diff
@@ -4556,6 +4556,7 @@
 				"ts-loader": "^6.2.1",
 				"tslint": "^5.20.1",
 				"typescript": "3.4.5",
+				"vscode-dotnet-runtime-library": "file:vscode-dotnet-runtime-library",
```

Comment:
I think this was left out from a previous change, just makes sure this directory is included in the packaged extension. It wasn't causing issues because package lock gets updated on build, but it should be committed.